.github/workflows/build_windows.yaml

```diff
@@ -0,0 +1,56 @@
+name: Build standalone tool for Windows
+
+on: 
+  push:
+    branches:
+      - master
+
+
+jobs:
+
+  Compile-on-Windows:
+
+    runs-on: windows-latest
+    
+    if: github.event_name == 'push' && startsWith(github.ref, 'refs/tags')
+
+    steps:
+
+      - uses: actions/checkout@v3
+      
+      - uses: robinraju/release-downloader@v1.8
+        with: 
+          repository: "MerginMaps/geodiff"    
+          latest: true
+          fileName: "geodiff_windows_binaries.zip"
+          zipBall: true
+          out-file-path: "scripts/windows_binaries"
+          extract: true
+      
+      - name: Delete files from binaries
+        run: |
+          cd scripts/windows_binaries
+          del geodiff_*.zip
+          del geodiff-*.zip
```

Comment:
why do we need to delete some stuff here?

Reply:
It is a result of previous step, the zipped file downloaded as artifact from geodiff which is unzipped and needs to be removed. It could probably stay there, but I am more comfortable with removing it before packing the exe file. With better download settings only one file needs to be removed now :)